src/main/python/dictfield.py

```diff
@@ -0,0 +1,25 @@
+from marshmallow.fields import Field
+
+
+class DictField(Field):
```

Comment:
Why not `Nested`?

Reply:
Because the format of json is such that keys are sweep numbers (that change)
    "sweep_features": {
      "11": {
        "spikes": [
          {

Reply:
https://marshmallow.readthedocs.io/en/stable/api_reference.html#marshmallow.fields.Dict?

Reply:
Actually Dict works, thanks.